docs/i18n.md

```diff
@@ -1,5 +1,121 @@
 # Internationalization
 
-To make text translatable, you’d use the `translate()` function from [i18n-calypso](../packages/i18n-calypso/README.md). We extract the translatable texts wrapped inside the function and write a POT file that is imported to [translate.wordpress.com](https://translate.wordpress.com/).
+If you add or modify text strings that will be displayed to users, consider whether they should be visible to non-English users.
+In that case, wrap the strings in the `translate()` function from [i18n-calypso](../packages/i18n-calypso/README.md).
+We extract the translatable texts wrapped inside the function and write a POT file that is exported to [translate.wordpress.com](https://translate.wordpress.com/).
 
 Translated texts are [split into chunks loaded via JSON asynchronously](translation-chunks.md).
+
+### String Freeze
+
+If you're launching or updating a feature, and want to make sure that all new strings are translated before merging,
```

Comment:
It might be a little more clear if you extend it a little explaining that adding the string freeze label would actually import the newly introduced strings to [WordPress.com Project in GlotPress](https://translate.wordpress.com/projects/wpcom/), making them available for translation prior to merging the changes to the trunk.

Reply:
I have added information based on our offline discussion.

Reply:
Thank you, It looks good!